0003/README.md

```diff
@@ -0,0 +1,123 @@
+---
+id: 0003
+title: Centralized Logging for Tinkerbell microservices.
+status: POC Done, Code change ready
+authors: Chitrabasu Khare <chitrabasukhare89@gmail.com>
+---
+
+## Summary:
+
+Centralized logging framework for Tinkerbell microservices. 
+
+## Overview: 
+
+- Tinkerbell will have the feature of centralized logging. 
+- Logging feature should be implemented in such a way that a user can modify the type of logging (rsyslog or fluentbit) as per his needs. This may require the user to make additional changes on his end to enable that feature. 
```

Comment:
The user should be able to modify the location where the log will be centralized. It does not need to be the provisioner

Reply:
User will have the flexibility to modify the log driver and log server as well. By default, the logs will be aggregated to provisioner.